### ISSUE TITLE
Menu scaling is really off in latest GZDoom dev build.

### ISSUE THREAD

**Issue by Accensus** [MatthewTheGlutton/HideousDestructor#209]
Can't even see the Exit option.

Probably related to https://forum.zdoom.org/viewtopic.php?f=7&t=70246.
Caused by HD having its own MENUDEF. Thread above has a solution. Haven't tried it, though.

**Comment by caligari87** [comment]
I'd wait until this is a problem in a stable release of GZDoom before worrying about it. Devbuilds are specifically noted as being unstable targets because they may have bugs, and it's unlikely Graf intended for this to happen.

**Comment by Accensus** [comment]
The thread has already been closed as [Won't fix] which is why I'm reporting it here instead of waiting on a fix from Graf.

**Comment by caligari87** [comment]
I posted in the thread to ask Graf if HD's MENUDEF should be changed, but of course I can't get a devbuild of GZDoom to run on this machine. Could you possibly post a screenshot in the thread showing the issue?

**Comment by Accensus** [comment]
Done.

**Comment by MatthewTheGlutton** [comment]
I don't see where the solution is. PR plz?

**Comment by MatthewTheGlutton** [comment]
I've reduced the number of options in the main menu to get around this in the meantime.

**Comment by MatthewTheGlutton** [comment]
Unless (a) someone wants the loadouts on the main menu again and (b) it won't push quit off the screen again I think I'll leave it as it is for now...

**Comment by MatthewTheGlutton** [comment]
Closing all issues on Github. This seems to be resolved anyway...
